### ISSUE TITLE
Direct users back to the original source of data

### ISSUE THREAD

**Issue by dpsnowden** [ioos/catalog#369]
Below is an email I received via the data.ioo@noaa.gov mailing list.  I think this points to a missing element in the UI.  Users discover data and expect an easy way to get it.  The pathway back to the original source is convoluted.  We need to make it very simple to get back to the original source of data.

> "My name is ..., and I am a Research Assistant with an environmental research consultancy for the government. I am currently working on a project to evaluate and project the economic impacts of ocean acidification on the shellfish industry in the Puget Sound region. As part of this effort, we are trying to compile ocean acidification measures from moorings in the region to help inform our calculations. While there appear to be relevant NANOOS datasets available via the IOOS catalog, I have been unable to connect to the necessary buoys and observation points thus far. Any help in this research would be greatly appreciated."

**Comment by emiliom** [comment]
Good point. Though what links to the "the original source" are appropriate and available may need some thinking. Or providing multiple links when available. Using SOS services as an example:
- A link to the DescribeSensor request for the dataset, which will return a SensorML XML that will be off-putting to many kinds of users (maybe a warning should be added, about what will be returned)
- operator, from SensorML: sml:member/sml:contact/ ... /sml:onlineResource
- publisher from SensorML: sml:member/sml:contact/ ... /sml:onlineResource
- webpage, from SensorML: sml:member/sml:documentation/ ... /sml:onlineResource

Just my 2 cents.
